webapp/lib/controller.dart

```diff
@@ -897,16 +897,11 @@ void sendReply(model.SuggestedReply reply, model.Conversation conversation) asyn
       translation: newMessage.translation,
       incoming: false);
   view.conversationPanelView.addMessage(newMessageView);
-  try {
-    await platform.sendMessage(conversation.docId, reply.text);
-  } catch (_) {
+  await platform.sendMessage(conversation.docId, reply.text, onError: (_) {
```

Comment:
Do we still need the `await`?

Reply:
If we omit `await` then it will not wait for the sendMessage future to complete and the exception will happen after we have exited the try/catch block.